packages/flutter/test/material/tooltip_test.dart

```diff
@@ -133,7 +133,7 @@ void main() {
      *********************/
 
     final RenderBox tip = tester.renderObject(find.text(tooltipText)).parent.parent.parent.parent.parent;
-    expect(tip.size.height, equals(20.0)); // 10.0 height + 5.0 padding * 2 (top, bottom)
+    expect(tip.size.height, equals(24.0)); // 14.0 height + 5.0 padding * 2 (top, bottom)
```

Comment:
Why did the height change by 4px?

Reply:
Because before, the font size was large enough to be constrained by the fixed-height container, and now it is allowed to grow to it's proper size.  The overall widget is still the correct size.

Reply:
Does the test also check the correct size?